mewarpx/mewarpx/utils_store/profileparser.py

```diff
@@ -0,0 +1,150 @@
+import sys
+import re
+import json
+
+
+class FullProfile(object):
+    """
+    Class for parsing, storing and writing the full profile data
+    """
+    def __init__(self, lines):
+        self.lines = lines
+        self.min = 0
+        self.avg = 0
+        self.max = 0
+        self.profile_dicts = []
+
+    def parse_full_profiling_output(self):
+        print("Parsing profiling output...")
+
+        self.exclusive_profiles = []
+        self.inclusive_profiles = []
+
+        # The first 5 lines are not needed for parsing
+        del self.lines[0:5]
+        index = 0
+        count = 0
+        for line in self.lines:
```

Comment:
Could you factor out this whole `for` loop into a separate function and then call it here and in place of its copy just below? I think it would just need to take a string `"incl"` or `"excl"` as an argument to use as a prefix for the metric names.

Reply:
Yep you're right that'll work well